src/renderer/pages/project/script_editor.vue

```
@@ -513,6 +514,22 @@
     ...props.mulmoValue,
     imageParams: updatedImageParams,
   });
-  emit("saveMulmo");
+  emit("formatAndPushHistoryMulmoScript");
+};
+
+const deleteReferenceImage = (imageKey: string) => {
+  const currentImages = props.mulmoValue?.imageParams?.images ?? {};
+  const { [imageKey]: _, ...updatedImages } = currentImages;
```

ESLint

[error] '_' is assigned a value but never used. Allowed unused vars must match /^__/u.

(@typescript-eslint/no-unused-vars)